### ISSUE TITLE
cannot run keepassxc-cli since a few days/weeks

### ISSUE THREAD

**Issue by lerignoux** [keepassxreboot/keepassxc#5701]
## Overview
On an up to date Manjaro, while it was working before, I now cannot run keepass-cli or keepass anymore.
It fails on the follow error:
```
keepassxc-cli: error while loading shared libraries: libquazip5.so.1: cannot open shared object file: No such file or directory
```


## Steps to Reproduce
1. install a manjaro system
2. update everything
3. install keepassxc-cli `pacman -Syu keepassxc`
4. run `keepassxc-cli -c`

## Expected Behavior
The CLI should run smoothly


## Actual Behavior
An crash is output


## Context
just updated my system.
I think maybe the library has been renamed ?
checking my lib64, I found the following packages
```
lau-lin:~/ $ ls -alh /lib64/*libquazi*                                                                                                                                                                                                                             [15:34:29]
lrwxrwxrwx 1 root root   23 10月 12 02:48 /lib64/libquazip1-qt5.so -> libquazip1-qt5.so.1.0.0
lrwxrwxrwx 1 root root   21 10月 12 02:48 /lib64/libquazip1-qt5.so.1.0.0 -> libquazip1-qt5.so.1.1
-rwxr-xr-x 1 root root 226K 10月 12 02:48 /lib64/libquazip1-qt5.so.1.1
-rwxr-xr-x 1 root root 226K 11月 15 15:23 /lib64/libquazip1.so
```
could it be they added the -qt5 ?

I tried to reinstall libquazip(-dev) to no avail.

[NOTE]: # ( Paste debug info from Help → About here )
```
lau-lin:~/ $ sudo pacman -Q  keepassxc                                                                                                                                                                                                                         
keepassxc 2.6.2-1
```
i cannot run it to fetch the version directly

[NOTE]: # ( Pick choices based on your environment )
Operating System: Linux
Desktop Env: Cinnamon
Windowing System: X11

**Comment by lerignoux** [comment]
Just remembered I compiled keepassxc for a feature this year, it should not affect the current issue on the official package, but` just in case.

**Comment by droidmonkey** [comment]
You need to install the correct QuaZip package. They updated recently and made a lot of problems along with it. You need to 0.9 version.

**Comment by lerignoux** [comment]
> You need to install the correct QuaZip package. They updated recently and made a lot of problems along with it. You need to 0.9 version.

Thank you a lot indeed, it solved my issue !

**Comment by mareksamec** [comment]
Thanks, on Arch linux I solved it just by installing extra/quazip 1.1-1 from AUR instead of the older version 0.7.2 I had after system update.
